101-lab/content/09_persistent_configurations.md

```diff
@@ -56,7 +56,7 @@ metadata:
 <kbd>![](./images/07_persistent_config_06.png)</kbd>
 
 #### Changing Config Map Content
-The content in your `configMap` can be changed and in most cases it's NOT dynamically updated in the pod. With that said, a new deployment will be required for the changes to be picked up.
+When a `configMap` is referenced by a deployment that uses the `rolling` deployment strategy, any changes to that `configMap` will be automatically applied to the pods created by that deployment. <!-- #TBC If your deployment uses the `recreate` strategy, the changes from your configmap will apply when you next roll out new pods.  -->
```

Comment:
I am not sure about the second part yet. But first part about rolling deployment is correct. And we should mention that this feature comes on 4.12 feature.

Reply:
Okay, is there any documentation that I can point to that articulates this change? I couldn't find anything about this in the [4.12 release notes ](https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html)

Reply:
My bad, I counldn't find the evidence I read earlier, but here's the doc for this: https://kubernetes.io/docs/concepts/configuration/configmap/#mounted-configmaps-are-updated-automatically